test/integration_test/basic_test.go

```diff
@@ -100,6 +98,20 @@ var testStorageClusterBasicCases = []types.TestCase{
 					Enabled: true,
 				},
 			}
+			trueVal := true
+			cluster.Spec.Storage = &corev1.StorageSpec{
+				UseAll:               nil,
+				UseAllWithPartitions: &trueVal,
+				ForceUseDisks:        &trueVal,
+				Devices:              nil,
+				CacheDevices:         nil,
+				JournalDevice:        nil,
+				SystemMdDevice:       nil,
+				KvdbDevice:           nil,
```

Comment:
Just curious. Why do we need to set the rest of them to nil?

Reply:
it's auto generated.

Reply:
But we can remove it from the code, right?